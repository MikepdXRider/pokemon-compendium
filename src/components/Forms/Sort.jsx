import React from 'react'

export default function Sort({setSelectedSortOrderStr, selectedSortOrderStr, optionsArr}) {
    return (
        <label>
            Sort
            <select value={selectedSortOrderStr} onChange={(e) => setSelectedSortOrderStr(e.target.value)}>
                <option value="None">None</option>
                {
                    optionsArr.map(option => <option value={option} key={option}>{option}</option>)
                }
            </select>
        </label>
    )
}


