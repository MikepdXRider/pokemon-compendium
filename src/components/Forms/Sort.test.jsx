import { render } from "@testing-library/react";
import Sort from "./Sort.jsx";

const optionsArr = ['test-options1', 'test-options2'];

it('matches snapshot', () => {
    const {container} = render(<Sort optionsArr={optionsArr}/> );
    expect(container).toMatchSnapshot();
})