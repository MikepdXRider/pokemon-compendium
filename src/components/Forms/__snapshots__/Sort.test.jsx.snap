// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <label>
    Sort
    <select>
      <option
        value="None"
      >
        None
      </option>
      <option
        value="test-options1"
      >
        test-options1
      </option>
      <option
        value="test-options2"
      >
        test-options2
      </option>
    </select>
  </label>
</div>
`;
